requirements.txt

```diff
@@ -0,0 +1,11 @@
+--only-binary=numpy
+--only-binary=scipy
+--only-binary=matplotlib
+docopt
+psutil
+nose
+coverage
+#coveralls?
+#requests?
+#deprecation?
+#nibabel?
```

Comment:
Probably we want nibabel for testing - it is optional. It seems the preferred way to do this is an optional-requirements.txt. I can add this?

Reply:
ok. `coveralls` is used for coverage testing as well I believe, so optional. I thought we needed `deprecation` but that should be easy to check. `requests` I have no idea